Dfe.Data.SearchPrototype/Web/Tests/PartialIntegration/HomeControllerTests.cs

```diff
@@ -79,7 +84,7 @@ public async Task Index_WithNoResults_ReturnsNoSearchResultsOnModel()
         var viewResult = Assert.IsType<ViewResult>(result);
         var viewModel = Assert.IsType<Models.ViewModels.SearchResults>(viewResult.Model);
 
-        viewModel.SearchItems.Should().BeEmpty();
+        viewModel.SearchItems.Should().BeNull();
```

Comment:
what's the impact of changing this behaviour?

Reply:
... answer is nothing - it's dealt with in the view model

Reply:
Yeah, it seemed appropriate given the way it was being handled in the model, ok to resolve?